src/App.js

```diff
@@ -5,30 +5,33 @@ import SideBar from './components/SideBar';
 import LoginRegister from './components/LoginRegister';
 import { HStack, Heading } from '@chakra-ui/layout';
 import '@fontsource/poppins';
+import AuthContextProvider from './components/AuthContextProvider';
 
 const App = () => {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [activeOption, setActiveOption] = useState('login');
```

Comment:
We can now refactor this part to use the router instead of conditional rendering

- Create two routes, `/login` and `/register`, and separate Login and Register page components, so we can render each on an appropriate route